.stylish-haskell.yaml

```diff
@@ -0,0 +1,236 @@
+# stylish-haskell configuration file
+# ==================================
```

Comment:
I  like `stylish-haskell` because it's a relatively unopiniononated way of formatting the most annoying parts of a codebase - imports, exports, and language pragmas.